power_whisperpower.orogen

```diff
@@ -45,6 +50,38 @@ task_context "DCPowerCubeTask" do
     port_driven
 end
 
+# Driver component for the PMG Genverter
+task_context "PMGGenverterTask" do
+    needs_configuration
+
+    # The duration sending the restart command before the start command
+    #
+    # This is done to ensure that the device is fully reseted before the startup
```

Comment:
No need to fix that, but `reseted` does not exist, it's `reset` all the way (I have reset)